src/main/java/mate/academy/dao/jdbc/CarJdbcDao.java

```diff
@@ -0,0 +1,203 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import mate.academy.dao.CarDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Car;
+import mate.academy.model.Driver;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class CarJdbcDao implements CarDao {
```

Comment:
```suggestion
public class CarJdbcDaoImpl implements CarDao {
```
same for other DaoImpl classes